for-developers/services/task-service.md

```diff
@@ -106,6 +106,129 @@ If AWS Lambda does not report anything within 5 minutes, the associated task sta
 There is no automatic retrying for a failed task. To rerun a failed Task, create a new Task with the same parameters
 {% endhint %}
 
+## Tasks Service Functions API&#x20;
```

Comment:
What does the `&#x20;` mean?

Reply:
Maybe also mention here from which Task service version this functionality is available.